api/app/controllers/api/v1/applicants_controller.rb

```diff
@@ -16,6 +16,11 @@ def index
 
         # POST /applicants
         def create
+            # if we passed in an id that exists, we want to update
+            if params[:id] && Applicant.exists?(params[:id])
+                update and return
+            end
+
             applicant = Applicant.new(applicant_params)
             if applicant.save # passes Applicant model validation
                 render_success(applicant)
```

Comment:
For line 34 in this file, couldn't we techinically hit PUT/PATCH on applicants/:id?

Reply:
Yes, we could do that. I guess I'll just remove that if statement.